tasks/queue-granules/schemas/config.json

```diff
@@ -15,6 +15,8 @@
     "internalBucket": { "type": "string" },
     "provider": { "type": "object" },
     "queueUrl": { "type": "string" },
-    "stackName": { "type": "string" }
+    "stackName": { "type": "string" },
+    "state_machine": { "type": "string" },
```

Comment:
I thought we had talked about adding these via the message adapter? Is that coming? I think that makes more sense rather than to have the user configure from `cumulus_meta`

Reply:
I thought we will add them to message adapter later time.  If we add to message adapter, are we adding them to input config or a new property like cumulus_meta?

Reply:
That's a good question. Maybe see if anyone has any ideas? I was thinking config, but then we'd have to add it after schema validation. I'm also fine with a cumulus_meta. I don't have a solid idea here.